plugins/tekton/src/components/PipelineRunList/PipelineRunRowActions.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+
+import { Flex, FlexItem } from '@patternfly/react-core';
+import { Tooltip } from '@patternfly/react-core/dist/esm/components/Tooltip/Tooltip';
+
+import { PipelineRunKind } from '@janus-idp/shared-react';
+
+import { TektonResourcesContext } from '../../hooks/TektonResourcesContext';
+import { getSbomTaskRun } from '../../utils/taskRun-utils';
+import ViewLogsIcon from '../Icons/ViewLogsIcon';
+import PipelineRunLogDialog from '../PipelineRunLogs/PipelineRunLogDialog';
+import PipelineRunSBOMLink from './PipelineRunSBOMLink';
+
+const PipelineRunRowActions: React.FC<{ pipelineRun: PipelineRunKind }> = ({
+  pipelineRun,
+}) => {
+  const { watchResourcesData } = React.useContext(TektonResourcesContext);
+  const [open, setOpen] = React.useState<boolean>(false);
+  const [noActiveTask, setNoActiveTask] = React.useState(false);
+  const pods = watchResourcesData?.pods?.data || [];
+  const taskRuns = watchResourcesData?.taskruns?.data || [];
+  const sbomTaskRun = getSbomTaskRun(pipelineRun?.metadata?.name, taskRuns);
+  const activeTaskName = sbomTaskRun?.metadata?.name;
+
+  const openDialog = (viewLogs?: boolean) => {
+    if (viewLogs) setNoActiveTask(true);
+    setOpen(true);
+  };
+
+  const closeDialog = () => {
+    setNoActiveTask(false);
+    setOpen(false);
+  };
+
+  return (
+    <>
+      <PipelineRunLogDialog
+        pipelineRun={pipelineRun}
+        open={open}
+        closeDialog={closeDialog}
+        pods={pods}
+        taskRuns={taskRuns}
+        activeTask={noActiveTask ? undefined : activeTaskName}
+      />
+      <Flex gap={{ default: 'gapSm' }}>
+        <FlexItem>
+          <Tooltip content="View logs">
+            <ViewLogsIcon onClick={() => openDialog(true)} />
```

Comment:
I think just for accessibility reasons alone it'd be better to wrap this icon in a button with plain styling and attach the event handling to that.

Reply:
wrapped both icons in IconButton component and attached onclick handler to it.